### DISCUSSION TITLE
Add an option to inline all libraries used in a given DSP

### DISCUSSION THREAD

**Discussion by sletz** [grame-cncm/faust#832] [Ideas]
- So that to obtain a self-contained file. Note that with would be different from the current` -e` option that exports an expanded but somewhat unreadable DSP file.
-  Recursive library import must be implemented
- The code used to handle the `-i ` option can probably be reused for that

**Comment by dfl** [comment]
attempted this here https://github.com/dfl/faust/tree/feature/inline-libraries
